src/pages/RFOX/hooks/useEarnedQuery.ts

```diff
@@ -1,4 +1,5 @@
-import { skipToken } from '@tanstack/react-query'
+import { skipToken, useQuery } from '@tanstack/react-query'
+import type { ReadContractQueryKey } from '@wagmi/core/query'
```

Comment:
`@wagmi/core/query` vs. `wagmi/query` previously as referenced in the [wagmi docs](https://wagmi.sh/core/api/actions/readContract#tanstack-query)